src/material-experimental/mdc-menu/testing/menu-harness.ts

```diff
@@ -6,12 +6,20 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ComponentHarness, HarnessPredicate, TestElement, TestKey} from '@angular/cdk/testing';
+import {
+  ComponentHarness,
+  ContentContainerComponentHarness,
+  HarnessLoader,
+  HarnessPredicate,
+  HarnessQuery,
+  TestElement,
+  TestKey,
+} from '@angular/cdk/testing';
 import {coerceBooleanProperty} from '@angular/cdk/coercion';
 import {MenuHarnessFilters, MenuItemHarnessFilters} from '@angular/material/menu/testing';
 
 /** Harness for interacting with an MDC-based mat-menu in tests. */
-export class MatMenuHarness extends ComponentHarness {
+export class MatMenuHarness extends ContentContainerComponentHarness<string> {
```

Comment:
Should we `ContentContainerComponentHarness` rather? (for all similar harnesses too). That would help us enforce the methods being properly overridden and a consistent signature/API. Also looks like the base class isn't used at runtime anyway.

Reply:
I'll do a follow-up to try and make it so we don't have to override these methods, but we pass in something through the constructor instead.

Reply:
Sounds great 👍